src/components/ChangePlan/PlanCalculator.js

```diff
@@ -181,6 +180,13 @@ const PlanPricePerMonth = ({ planData }) => {
 const PlanAgreement = ({ planData }) => {
   const intl = useIntl();
   const _ = (id, values) => intl.formatMessage({ id: id }, values);
+  const discountPercentage = planData.discount?.discountPercentage;
```

Comment:
Could you add some test for these components that were modified separately? `PlanAgreement` `PlanPricePerMonth`  `PlanPriceWithoutDiscounts`

Reply:
I could create a new ticket to refactor this component with the main goal to move this components to new files. Then we could have separated tests for each one. What do you think?

Reply:
Ticket to do this: [DW-990](https://makingsense.atlassian.net/browse/DW-990)